pkgs/tools/misc/hdf5/default.nix

```diff
@@ -103,6 +103,13 @@ stdenv.mkDerivation rec {
       mv "$file" "''${file%%-shared}"
     done
     popd
+  '' + lib.optionalString fortranSupport
+  ''
+    mkdir -p $dev/include
+    cp $out/mod/shared/* $dev/include
```

Comment:
Could these two lines not simply be a `mv $out/mod/shared/* $dev/include`?

Otherwise very nice improvement.

Reply:
Yes, it can be replaced. Thanks.